source/js/scripts/app.js

```diff
@@ -136,6 +136,12 @@ function updateAppTitle(taskFinished) {
   const appSubtitle = document.querySelector('.app-subtitle');
 
   let subtitle = '';
+  let taskName = taskList.todo[0].name;
```

Comment:
You need to generate the taskname after we check if we're at the end of a session (down to line 161~)
Otherwise the code tries to read the empty tasklist and breaks

Reply:
should be fixed now